README.md

```diff
@@ -2,22 +2,16 @@ Extension Written by Scarlet#1115 on Discord.
 
 **Please do not redistribute yet... This is stil in Beta.**
 
-### Installation
-1. Download/clone this repo
-2. In Chrome, open [chrome://extensions](chrome://extensions), ensure Developer Mode is on (top right toggle).
-3. In the top left, click "Load unpacked"
-4. Navigate to the ./dist of this repo. 
-5. Click "Select" when in the folder.
-6. Navigate to https://ilmina.com/#/Valeria
-
 ### Building from source
 1. Make sure you have npm installed
 2. Open a terminal in this repo and run `npm install`
 3. Run `npm run build` to create an build in ./dist
-4. Navigate to https://ilmina.com/#/Valeria
+
+### Running local copy
+Currently Valeria is a static site, so you just need to host those static files. You can run `npm run serve-dist` to host it locally. Valeria is probably available in https://localhost:5000
```

Comment:
Change to port 4800 as that is Valeria's ID number

Reply:
done